docs-ref-conceptual/eclipse/toc.yml

```diff
@@ -1,19 +1,19 @@
 - name: Azure Toolkit for Eclipse
   href: azure-toolkit-for-eclipse.md
   items:
-  - name: Install 
-    href: azure-toolkit-for-eclipse-installation.md
   - name: Get Started
     href: azure-toolkit-for-eclipse-create-hello-world-web-app.md
-  - name: Tutorials
+  - name: Step-by-Step Tutorials
```

Comment:
I guess we cannot change "Tutorials" to " Step-by-Step Tutorials". See other TOCs. "Get Started", "Tutorials" and "How-to" are common categories.